utils/validateSearchParams.ts

```diff
@@ -0,0 +1,22 @@
+import { NEWS_TAGS, NOTICE_TAGS } from '@/constants/tag';
+
+export const validatePageNum = (pageNumStr: string) => {
+  const pageNum = Number.parseInt(pageNumStr);
+  const isValid = Number.isNaN(pageNum) || pageNum < 1 ? false : true;
```

Comment:
NOT을 나타낼 때는 삼항 연산자보다 NOT 연산자가 더 명확한 것 같은데 어때?

```suggestion
  const isValid = !(Number.isNaN(pageNum) || pageNum < 1)
```

그리고 개인적으로는 아래가 조건을 더 명확하게 보여주는 것 같음

```suggestion
  const isValid = !Number.isNaN(pageNum) && 1 <= pageNum
```

Reply:
굿굿 예전에 저렇게 짜두고 넘어갔나보다